apps/src/gamelab/GameLabVisualizationColumn.jsx

```diff
@@ -56,13 +59,29 @@ const SongSelector = Radium(class extends React.Component {
     }
   };
 
+  componentWillMount() {
+    fetch(`/api/v1/sound-library/hoc_song_meta/songManifest.json`)
+      .then((response) => {return response.json();})
+      .then((data) => {this.parseSongOptions(data);});
+  }
+
+  parseSongOptions(data) {
+    let songs = {};
+    if (data) {
+      data.songs.forEach((song) => {
+        songs[song.id] = {title: song.text, url: song.url};
+      });
```

Comment:
It looks like there's some repeated fetch/parsing logic here - it'd be nice to deduplicate this.

Reply:
It looks like the old-runtime is going to be deprecated and removed, so this whole file will be removed soon.